.circleci/template.yml

```diff
@@ -442,7 +442,7 @@ jobs:
       - run:
           name: Build GraphQL static docs
           command: |
-            npm install --prefix=$HOME/.local --global spectaql
+            npm install --global cheerio@1.0.0-rc.12 spectaql
```

Comment:
Because of it, the following line, `npx` didn't use the preinstalled packages and reinstalled them once more: https://app.circleci.com/pipelines/github/esl/MongooseIM/12908/workflows/4640d824-6f90-4bcf-bbfe-2832919abbfa/jobs/234987?invite=true#step-103-19
And I didn't find a way to pin a transitive dependency using `npx` itself. But this behaviour was clearly not intended anyway.